src/configure/templateInputHelper/RepoAnalysisSettingInputProvider.ts

```diff
@@ -71,10 +88,16 @@ export class RepoAnalysisSettingInputProvider {
                 else {
                     selectedValue = await new ControlProvider().showQuickPick(repoAnalysisSettingKey, possibleValues, { placeHolder: inputControl.getInputDescriptor().name });
                 }
-
                 if (settingIndexMap.get(selectedValue.data).length === 1) {
                     this._selectedRepoAnalysisSettingIndex = settingIndexMap.get(selectedValue.data)[0];
```

Comment:
this also need to be set in case of array

Reply:
this is for setting up repository analysis setting in use.